CMakeLists.txt

```diff
@@ -14,14 +14,14 @@ if (WIN32)
 	set(KIMAGEANNOTATOR_LANG_INSTALL_DIR "translations")
 
 	set(CPACK_GENERATOR WIX)
```

Comment:
I think this whole section with CPACK variables should go into the `src/CMakeLists.txt`, there we have already a similar section for MacOS.

Reply:
can you fix this?